src/js/components/Box/propTypes.js

```diff
@@ -153,6 +153,7 @@ if (process.env.NODE_ENV !== 'production') {
         'xlarge',
       ]),
       PropTypes.string,
+      PropTypes.object,
```

Comment:
```suggestion
      PropTypes.shape({ row: PropTypes.string, column: PropTypes.string }),
```
also just being more specific

Reply:
good idea